scripts/generate_screenshots/apply_variant.sh

```
@@ -0,0 +1,114 @@
+  fi
+
+  lang=$(basename "$lang_dir")
+  dest_dir="$FASTLANE_DIR/$lang"
+  mkdir -p "$dest_dir"
+
+  # 適用先に残った過去の PNG と混ざった一式を作らないよう、適用言語の既存 PNG を先に削除する
+  find "$dest_dir" -maxdepth 1 -name '*.png' -delete 2>/dev/null || true
+
+  # バリアントのスクリーンショットを適用先にコピー
+  cp -f "$lang_dir"*.png "$dest_dir/" 2>/dev/null || true
+  file_count=$(ls -1 "$dest_dir"/*.png 2>/dev/null | wc -l | tr -d ' ')
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)